exports/lint-configs/.clang-format

```
@@ -101,6 +104,7 @@ LineEnding: "LF"
 MainIncludeChar: "Quote"
 MaxEmptyLinesToKeep: 1
 NamespaceIndentation: "None"
+OneLineFormatOffRegex: "^// NOLINT"
```

```suggestion
OneLineFormatOffRegex: "^// clang-format offnextline"
```

This text is more clear that it is clang-format being skipped and matches the multi-line clang format syntax.